SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -232,24 +281,44 @@ install -D -m 0755 %{_builddir}/%{name}-%{version}/tools/osbuilder/image-builder
 %{osbuilder}/modules/*
 %{osbuilder}/tools/*
 
-# Remove some scripts we don't use
+# remove some scripts we don't use
 %exclude %{osbuilder}/tools/osbuilder/rootfs-builder/alpine
 %exclude %{osbuilder}/tools/osbuilder/rootfs-builder/centos
 %exclude %{osbuilder}/tools/osbuilder/rootfs-builder/clearlinux
 %exclude %{osbuilder}/tools/osbuilder/rootfs-builder/debian
 %exclude %{osbuilder}/tools/osbuilder/rootfs-builder/template
 %exclude %{osbuilder}/tools/osbuilder/rootfs-builder/ubuntu
 
-
 %changelog
+*   Mon Sep 18 2023 Dallas Delaney <dadelan@microsoft.com> 0.6.1-1
+-   Pull in latest source for genpolicy, utarfs, and overlay changes
```

Comment:
We should be able to remove the alpha/beta package changelogs here. @ms-mahuber thoughts?

Reply:
I went ahead and removed alpha/beta changelogs.